Elevel/Elevel.Api/appsettings.json

```diff
@@ -6,8 +6,17 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
+
   "AllowedHosts": "*",
+
   "ConnectionStrings": {
     "DefaultConnection": "Server=PC\\SQLEXPRESS;Database=Elevel;Trusted_Connection=True;MultipleActiveResultSets=true"
+  },
+
+  "JWT": {
+    "key": "C1CF4B7DC4C4175B6618DE4F55CA4",
+    "Issuer": "SecureApi",
+    "Audience": "SecureApiUser",
+    "DurationInMinutes": 65
   }
 }
```

Comment:
Why don't you use SQL client?

Reply:
This parameter doesn't need here. EF will do that.